### ISSUE TITLE
Too many list items created from init text consisting of multiple lines

### ISSUE THREAD

**Issue by OmarHawk** [samclarke/SCEditor#359]
For some reasons, too many list items are created, when creating a list from text consisting of multiple lines. In my case, this happens only when the text was already in the editor during initialization of the editor. This occurs in Firefox only (I use Firefox 33.0.2 and tested also Chrome).

Captured Screen:
![rec_36](https://cloud.githubusercontent.com/assets/2911492/4919856/b2e15c26-64fd-11e4-920d-a5728724e3c5.gif)
JS-Fiddle: http://jsfiddle.net/jprheyfh/

I am using the latest dev version.

**Comment by samclarke** [comment]
Closing as this seems to have already been fixed.

**Comment by OmarHawk** [comment]
@samclarke : This has not been fixed. I have update the fiddle with the latest version, and it still has the exact same behaviour:

http://jsfiddle.net/jprheyfh/1/

**Comment by samclarke** [comment]
Whoops, for some reason I read it as a Chrome issue not a FF issue. Reopening.

**Comment by samclarke** [comment]
I've been looking into this and after a lot of testing it looks like it isn't `<br>` that's causing the issue after all. FF also adds extra lines to:
```html
<p>test</p>
<p>test</p>
```
which becomes:
```html
<li>test</li>
<li><br type="_moz"></li>
<li>test</li>
```
This issue seems to be caused by FF wrapping the invisible newlines between the tags in an `li` element: https://jsfiddle.net/zr90y0tt/ so looks this is another FF bug.

Fixing it will require finding and removing the invisible newlines between the tags.

**Comment by samclarke** [comment]
Will be fixed in the 2.0.0 is release.
